src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -305,7 +301,7 @@ export const PublishingAccordion = ({
           </>
         )}
 
-        {lastPublishingRequest && !hasMismatchingPublishedStatus && (
+        {lastPublishingRequest && !hasMismatchingPublishedStatus && !isWaitingForFileDeletion && (
```

Comment:
Denne er vel en del av `hasMismatchingPublishedStatus` nå?

```suggestion
        {lastPublishingRequest && !hasMismatchingPublishedStatus && (
```